ui/Screen/Profile/Funding.svelte

```diff
@@ -0,0 +1,39 @@
+<script lang="ts">
+  import { wallet, Status } from "../../src/wallet";
+  import * as pool from "../../src/funding/pool";
+
+  import ConnectWallet from "../../DesignSystem/Component/Wallet/Connect.svelte";
+  import WalletPanel from "../../DesignSystem/Component/Wallet/Panel.svelte";
+
+  import Pool from "../Funding/Pool.svelte";
+
+  // Hack to have Svelte working with checking the $wallet variant
+  // and thus be able to access its appropriate fields.
+  $: w = $wallet;
```

Comment:
No, it's a hack so that svelte can realize the fine type it is handling. On line 27 we do:

``` svelte
{#if w.status === Status.Connected}
```

The scope within that condition has a `Connected` wallet at hand, but svelte is not able to figure that out and understand that `$wallet.connected.account` is an available field. It can't because we are referencing it as a store (`$wallet`). 

This hack basically just creates an alias to the store so that we can have Svelte figuring the fine type out while still working with the store. Another way to achieve this would be by casting the wallet as `any`, but that would blind us and easily lead to bugs and problems, so I figured this to be a better solution.